### DISCUSSION TITLE
Change default cursor

### DISCUSSION THREAD

**Discussion by martin-dimi** [xyflow/xyflow#2590] [Q&A]
Is it possible to change the default cursor of the pane? I've tried updating it via the `style` prop but with no luck:
```
 <ReactFlow
      style={{ cursor: "help" }}
...
```

I only need 1 interaction: Nodes to be selectable. No dragging, no panning, etc

PS: noticed I can hold shit and the modified cursor is now displayed as expected but I can no longer select nodes. Is there an option to make this behaviour default (multiselection) + being able to select nodes on click?

**Comment by bcakmakoglu** [comment]
You could try using simple css, so something like this

```css
.react-flow__pane {
  cursor: help;
}
```
